README.md

```diff
@@ -260,26 +231,15 @@ Failures:
 
 That's a weird looking "undefined method" error. It says we have no `author=` method.
 
-What? That's gotta be a mistake, right? Nope.
-
-Let's step back for a second. That `title` method we wrote before is what is called
-a "getter". We call it this because, well, it "gets" a property for us. But what
-happens if we want to, say, *set* a property, or an instance variable? This is where
+Let's step back for a second. That `title` method we wrote before is what is called a "getter". We call it this because it "gets" a property for us. But what happens if we want to, say, *set* a property? This is where
 "setters" come into play.
 
-Setters are methods that allow us to set instance variables. It seems weird, but they
-all are named like this: `property=`. This is because Ruby gives us a nice bit of
-syntactic sugar that allows us to use these methods like:
+Setters are methods that allow us to set instance variables. They are named like this: `property=`. This is because Ruby gives us a nice bit of syntactic sugar that allows us to use these methods like:
```

Comment:
Aren't they generally called attributes in ruby? I could be wrong. Properties I thought were the Obj-C term